Security.Platform.iOS/AgsSec.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using AeroGear.Mobile.Core.Utils;
+using AeroGear.Mobile.Security;
+
+namespace Security.Platform
+{
+    public class AgsSec
```

Comment:
This and the android implementation of this will probably share a common base class.
Too early now to create it, I will create as soon as I will add some more code.